### DISCUSSION TITLE
adding units to measurements

### DISCUSSION THREAD

**Discussion by Stevenv24** [babybuddy/babybuddy#917] [Ideas]
Hello, my wife and I have been using this for about a week, and we love it. Thank you for your hard work in making a great program that helps parents keep up.

Is there a way to add units of measurement to the measurements?

**Comment by cdubz** [comment]
See #689 for discussion around this. The lack of units is intentional, but I am more open to it now than I have been in the past (see https://github.com/babybuddy/babybuddy/issues/689#issuecomment-1673469327) -- just haven't had any time to spend really looking at it.

**Comment by brystmar** [comment]
Ideally the units would be configurable by the user.

My wife and I both agreed to use metric units for our first child, and it's just so much better in every way.  Imperial units are the worst -- stop making me use fractions!

**Comment by EcN000** [comment]
+1 to adding units. People who use these apps like seeing data, no? That way we can see growth charts and also see percentiles according to WHO database.

Coming from using apps like babyconnect or huckleberry, this is a much wanted feature. 

As pervious comment, it'd be nice to have configurable units and just convert in the back end and have some graphs according to percentile.

**Comment by BradNut** [comment]
Any decision here? Otherwise i guess we just have to be aware of using metric or imperial and be consistent. Not ideal but not breaking.
